src/scripts/main.js

```diff
@@ -1,7 +1,95 @@
 'use strict';
 
 // Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
+const game = new Game();
 
-// Write your code here
+const startBtn = document.querySelector('.button.start');
+// game.start();
+
+document.addEventListener('keydown', () => {
+  if (game.getStatus() !== 'playing') {
+    return;
+  }
+
+  let moveMade = false;
+
+  switch (event.key) {
+    case 'ArrowLeft':
+      game.moveLeft();
+      moveMade = true;
+      break;
+    case 'ArrowRight':
+      game.moveRight();
+      moveMade = true;
+      break;
+    case 'ArrowUp':
+      game.moveUp();
+      moveMade = true;
+      break;
+    case 'ArrowDown':
+      game.moveDown();
+      moveMade = true;
+      break;
```

Comment:
Create variables for event key words and use it here, also, add the blank line between the cases
```suggestion
    case 'ArrowLeft':
      game.moveLeft();
      moveMade = true;
      break;
      
    case 'ArrowRight':
      game.moveRight();
      moveMade = true;
      break;
      
    case 'ArrowUp':
      game.moveUp();
      moveMade = true;
      break;
      
    case 'ArrowDown':
      game.moveDown();
      moveMade = true;
      break;
```